app/src/ui/branches/pull-request-list.tsx

```diff
@@ -28,14 +29,34 @@ interface IPullRequestListProps {
   /** The pull requests to display. */
   readonly pullRequests: ReadonlyArray<PullRequest>
 
+  /** The currently selected pull request */
+  readonly selectedPullRequest: PullRequest | null
+
+  /** The name of the repository. */
+  readonly repositoryName: string
+
+  /** Is the default branch currently checked out? */
+  readonly isOnDefaultBranch: boolean
+
+  /** The current filter text to render */
+  readonly filterText: string
+
   /** Called when the user clicks on a pull request. */
   readonly onItemClick: (pullRequest: PullRequest) => void
 
   /** Called when the user wants to dismiss the foldout. */
   readonly onDismiss: () => void
 
-  readonly selectedPullRequest: PullRequest | null
+  /** Callback to fire when the filter text is changed */
+  readonly onFilterTextChanged: (filterText: string) => void
+
+  /** Called when the user opts to create a branch */
+  readonly onCreateBranch: () => void
+
+  /** Called when the user opts to create a pull request */
+  readonly onCreatePullRequest: () => void
 
+  /** Callbacked fired when user selects a new pull request */
```

Comment:
`s/Callbacked/callback/`

Reply:
Fixed in [62f9a3b](https://github.com/desktop/desktop/pull/3972/commits/62f9a3b9cfcede4c2fbcf10d8c47ed505752d99b)